test/integration_test.go

```diff
@@ -31,6 +31,8 @@ import (
 	"testing"
 	"time"
 
+	"go.uber.org/cadence/test/replaytests"
+
```

Comment:
huh.  I wonder why this file isn't being auto-goimports'd...  CI allowed it though so there's apparently a gap